icechunk-python/src/lib.rs

```diff
@@ -22,11 +24,20 @@ use errors::{
     IcechunkError, PyConflictError, PyConflictErrorData, PyRebaseFailedData,
     PyRebaseFailedError,
 };
+use icechunk::initialize_tracing;
 use pyo3::prelude::*;
 use repository::{PyGCSummary, PyRepository, PySnapshotInfo};
 use session::PySession;
 use store::PyStore;
 
+#[pyfunction]
```

Comment:
Should we have no logs by default instead?

Reply:
Yes I think that's "standard" behaviour? Particularly since we aren't plugged in to the python logging system.

Reply:
Though what are the available log levels? What is the default?

Reply:
foudn the answer here: https://docs.rs/tracing-core/0.1.33/tracing_core/metadata/struct.Level.html

Reply:
default is ERROR, available goes from trace to fatal iirc